### PR TITLE
Add support for theming, redux, and react-router

### PR DESCRIPTION
**Pull request by abefernan**
**Description**
This PR adds redux and react-router support to bierzo-wallet and also removes some of the CSS now made redundant by the ThemeProvider. It also features a dummy welcome page.

**Screenshots**
The following screenshots showcase each of the available routes:

*"/welcome" route:*
image
*"/payment" route:*
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,15 +58,21 @@
...
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-prettier": "^3.0.1",
+    "history": "^4.7.2",
     "husky": "^1.3.1",
     "lerna": "^3.13.0",
     "lint-staged": "^8.1.4",
     "prettier": "^1.16.4",
     "react": "16.9.0-alpha.0",
     "react-dom": "16.9.0-alpha.0",
+    "react-redux": "^6.0.1",
```
Comment (apanizo) [inline package.json]: we should use version 7.

File: packages/sanes-chrome-extension/package.json [hunk 2]
```diff
@@ -42,19 +42,13 @@
...
-    "connected-react-router": "^6.3.1",
     "encoding-down": "^6.0.1",
     "final-form": "^4.12.0",
-    "history": "^4.7.2",
     "jspdf": "^1.5.3",
     "level-js": "^4.0.0",
     "medulas-react-components": "*",
     "memdown": "^3.0.0",
     "random-js": "^2.0.0-rc5",
-    "react-redux": "^6.0.1",
```
Comment (apanizo) [inline packages/sanes-chrome-extension/package.json]: Good job here.

File: package.json (hunk 1 above)
Comment (apanizo) [reply package.json]: Can you try to update it, and run `yarn start`?

Comment (abefernan) [reply package.json]: Fixed in [3498dd4](https://github.com/iov-one/ponferrada/pull/101/commits/3498dd4c951b81651c765c0a5bf2609f734ee4ca).

**Pull request merged**